packages/ckeditor5-alignment/src/utils.js

```diff
@@ -44,3 +51,63 @@ export function isDefault( alignment, locale ) {
 		return alignment === 'left';
 	}
 }
+
+/**
+ * Brings the configuration to the common form, an array of objects.
+ *
+ * @param {Array.<String|Object>} configuredOptions Alignment plugin configuration.
+ * @returns {Array.<Object>} Normalized object holding the configuration.
+ */
+export function normalizeAlignmentOptions( configuredOptions = [] ) {
+	return configuredOptions.map( normalizeOption );
+}
+
+// @private
+function normalizeOption( option, index, allOptions ) {
+	let optionObj;
+
+	if ( typeof option == 'string' ) {
+		optionObj = Object.assign( {}, { name: option } );
+	} else {
+		optionObj = Object.assign( {}, defaultOptions[ index ], option );
```

Comment:
Are you sure that those arrays are equal in length and have the same logic on the same indexes?

Also, I would suggest using: 

```
optionObj = {
	...someObject,
	...someOverride,
	name: 'foo'
}
```

Reply:
Done.